src/main/scala-2.12/logback.xml

```diff
@@ -0,0 +1,12 @@
+<configuration>
```

Comment:
3 logback files! :)

Reply:
This one could be deleted, they shoud be in the resources directory